kivy/uix/bubble.py

```diff
@@ -80,7 +80,7 @@ class BubbleButton(Button):
     Rather use this BubbleButton widget that is already defined and provides a
     suitable background for you.
     '''
-    pass
+    always_release = BooleanProperty(True)
```

Comment:
`on_press` would just sidestep the bug, the same way `always_release = True` would. We're looking for a solution which lets users set both `True` and `False` for `always_release`.